examples/config.yml

```diff
@@ -107,7 +107,7 @@ lighthouse:
 # Port Nebula will be listening on. The default here is 4242. For a lighthouse node, the port should be defined,
 # however using port 0 will dynamically assign a port and is recommended for roaming nodes.
 listen:
-  # To listen on both any ipv4 and ipv6 use "[::]"
```

Comment:
ohh I guess this is the zero syntax w/o a port number, but the `[` and `]` are optional?